content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -19,6 +19,18 @@ Attendees are encouraged to add comments on talking points asynchronously before
 We plan and track our day-to-day work on our [GitHub board](https://github.com/orgs/sourcegraph/projects/200/views/1).
 The project is separated into multiple views: **Current iteration**, **Next iteration** (both grouped by issue status) and **All iterations** (grouped by milestone).
 
+We use an intentional list of statuses on our project board:
+
+- `No Status`: Status is not assigned. These issues need triage.
```

Comment:
nit: Can we just bold these instead of marking them as `<code>`, which is not semantically correct?
```suggestion
- **No Status**: Status is not assigned. These issues need triage.
```

Reply:
Sure!

Reply:
Done.